content/en/serverless/aws_lambda/metrics.md

```diff
@@ -468,9 +468,16 @@ A Lambda function may launch many concurrent execution environments when traffic
 
 Distributions provide `avg`, `sum`, `max`, `min`, `count` aggregations by default. On the Metric Summary page, you can enable percentile aggregations (p50, p75, p90, p95, p99) and also [manage tags][12]. To monitor a distribution for a gauge metric type, use `avg` for both the [time and space aggregations][13]. To monitor a distribution for a count metric type, use `sum` for both the [time and space aggregations][13]. Refer to the guide [Query to the Graph][14] for how time and space aggregations work.
 
+### Understanding your metrics usage, volume, and pricing in Datadog
+
+Datadog provides granular information about the custom metrics you're ingesting, the tag cardinality, and management tools for your custom metrics within the [Metrics Summary page][15] of the Datadog app. You can view all serverless custom metrics under the 'Serverlesss' tag in the Distribution Metric Origin [facet panel][16]. You can also control custom metrics volumes and costs with [Metrics without Limits™][17]. 
```

Comment:
```suggestion
Datadog provides granular information about the custom metrics you're ingesting, the tag cardinality, and management tools for your custom metrics within the [Metrics Summary page][15] of the Datadog app. You can view all serverless custom metrics under the 'Serverless' tag in the Distribution Metric Origin [facet panel][16]. You can also control custom metrics volumes and costs with [Metrics without Limits™][17]. 
```
I'm guessing this is just a typo, but didn't want to change a tag directly without flagging it!